File: comportamiento/chainOfResponsibility/ejemplo.py
class Manejador:
    siguiente = None
    def set_next(self, m): self.siguiente = m
    def manejar(self, req):
        if self.siguiente: self.siguiente.manejar(req)

class Soporte(Manejador):
    def manejar(self, req):
        if req == "simple": print("Resuelto")
        else: super().manejar(req)

File: comportamiento/chainOfResponsibility/ejemplo.java
abstract class Manejador {
    protected Manejador sig;
    public void setSig(Manejador m) { this.sig = m; }
    public abstract void manejar(String req);
}
class Soporte extends Manejador {
    public void manejar(String req) {
        if(req.equals("simple")) System.out.println("Resuelto");
        else if(sig != null) sig.manejar(req);
    }
}